pkg/plugins/backendplugin/coreplugin/core_plugin.go

```diff
@@ -24,10 +26,11 @@ type corePlugin struct {
 
 // New returns a new backendplugin.PluginFactoryFunc for creating a core (built-in) backendplugin.Plugin.
 func New(opts backend.ServeOpts) backendplugin.PluginFactoryFunc {
-	return func(pluginID string, logger log.Logger, _ func() []string) (backendplugin.Plugin, error) {
+	return func(pluginID string, logger log.Logger, tracer tracing.Tracer, _ func() []string) (backendplugin.Plugin, error) {
 		return &corePlugin{
 			pluginID:            pluginID,
 			logger:              logger,
+			tracer:              tracer,
```

Comment:
Do we need tracer for corePlugin? We changed the `backend.PluginFactoryFunc` signature, if it is not needed we can omit the param `func(pluginID string, logger log.Logger, _ tracing.Tracer,` ..)

Reply:
We can probably omit it for now 👍